rollout/controller.go

```diff
@@ -341,15 +341,14 @@ func (c *Controller) syncHandler(key string) error {
 	if err != nil {
 		return err
 	}
-	log.WithField(logutil.RolloutKey, name).WithField(logutil.NamespaceKey, namespace).Infof("Started syncing rollout at (%v)", startTime)
 	rollout, err := c.rolloutsLister.Rollouts(namespace).Get(name)
 	if k8serrors.IsNotFound(err) {
-		log.WithField(logutil.RolloutKey, name).WithField(logutil.NamespaceKey, namespace).Info("Rollout has been deleted")
 		return nil
 	}
 	if err != nil {
 		return err
 	}
+	log.WithField(logutil.RolloutKey, name).WithField(logutil.NamespaceKey, namespace).Infof("Started syncing rollout at (%v)", startTime)
```

Comment:
This was previously logging rollouts that were enqueued that were not matching the instance id.